src/client/Microsoft.Identity.Client/Http/ExponentialRetryStrategy.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Identity.Client.Http
+{
+    internal class ExponentialRetryStrategy
```

Comment:
Do we really need these "strategy" objects? It seems to me that you can just put all the inputs in the `CalculateDelay` method and make that static and part of the Policy object.

Reply:
I made them their own objects so that they could be re-used across retry policies. I think it makes sense to keep them their own objects since a retry policy like IMDS's uses both exponential and linear strategy.